docs/time/index.rst

```diff
@@ -276,13 +276,18 @@ tdb    Barycentric Dynamical Time  (TDB)
 tt     Terrestrial Time            (TT)
 ut1    Universal Time              (UT1)
 utc    Coordinated Universal Time  (UTC)
+local  Local Time Scale            (LOCAL)
```

Comment:
Since this is typeset with the `(...)` part next to the text, maybe just right-adjust `(LOCAL)` to the end of the column (and move `(TT)` out by one space)